documentation/docs/pages/contributing/index.md

```diff
@@ -9,7 +9,7 @@ If you would like to interact in person with the development team, there are reg
 We also manage most of our development on GitHub. We track AM3 work on a [GitHub project board here](https://github.com/orgs/ACCESS-NRI/projects/35). If you'd like to have your issue considered for AM3 development, please [open an issue](https://github.com/ACCESS-NRI/access-am3-configs/issues/new) on the ACCESS-AM3 configurations repository.
 
 !!! warning
-    The ACCESS-AM3 configurations repository is private due to UK Met Office licensing restrictions. If you would like to get involved, please get in contact with us on the [Hive forum]() to request access. **[TODO] This is something we don't want public yet? Feels very contradictory to say "All contributions, please come get involved" and then say "but wait our repository is private"**
+    This model uses components and science configurations from the UK Met Office (UKMO), and as such are license restricted. This means the associated repositories are required to be private. If you would like to be granted access to these repositories, please get in touch **where?**.
```

Comment:
Edit: be consistent with naming, i.e., **ACCESS-Hive forum** and **ACCESS-AM3**. Sometimes they are referred to as "Hive forum" and "AM3"